app/views/applicants/new.html.erb

```diff
@@ -30,16 +30,23 @@
   <table  class="lined">
     <%= form_tag(new_project_applicant_path(project_id: params[:project_id]), method: :get, id: "username_form") do %>
       <tr>
-        <td class="title">PI Username</td>
+        <td class="title">
+          <%= "#{@project.applicant_abbreviation_wording.html_safe} Username" -%>
+        </td>
         <td>
           <%= text_field_tag :username, @applicant.username %>
           <%= link_to_function(image_tag("arrow_refresh.png",
                                style: "margin-left:10px; margin-bottom:-10px; margin-top:-5px;",
                                title: "click here to reload this page with a new #{NucatsAssist.institute_id_name}"),
                                "jQuery('#username_form').submit()") -%>
           <%# FIXME: Why does this only work for NU personnel? %>
-          Username <%= netid_lookup_tag %> for the Principal Investigator
-          <em>(currently this change PI feature works only for NU-afflilated personnel.)</em>
+          Username <%= netid_lookup_tag %> for the
+          <%= @project.applicant_wording.html_safe -%>
```

Comment:
Same here re: using safe navigation operator `&.`

Reply:
Thanks that was a good catch.